src/commons/theme.ts

```diff
@@ -0,0 +1,41 @@
+import Joi from 'joi';
+
+const colorSchema = Joi.object().keys({
+  primary: Joi.string().required().example('#ff00ff'),
```

Comment:
We could consider adding in a .regex() checking against something like this ^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$

Reply:
Haaaa, yes, I wanted to use `.hex()` but it was not exactly what I wanted. I forgot/was not aware of `.regexp()`, I'll do this change.
Thanks 🙏

Reply:
a3437c9d9be60d60aaa3f7273d657ebb18f3b469